src/app/core/services/platform/v1/shared/expenses.service.ts

```
@@ -80,15 +83,15 @@ export class ExpensesService {
         paymentMode: this.getPaymentModeForExpense(expense, paymentModes),
       }))
       .reduce((paymentMap: PaymentModeSummary, expenseData) => {
-        if (paymentMap.hasOwnProperty(expenseData.paymentMode.key)) {
-          paymentMap[expenseData.paymentMode.key].name = expenseData.paymentMode.name;
-          paymentMap[expenseData.paymentMode.key].key = expenseData.paymentMode.key;
-          paymentMap[expenseData.paymentMode.key].amount += expenseData.amount;
-          paymentMap[expenseData.paymentMode.key].count++;
+        if (paymentMap.hasOwnProperty(expenseData.paymentMode.key as string)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)